models/public/mobilefacedet-v1-mxnet/mobilefacedet-v1-mxnet.md

```diff
@@ -0,0 +1,100 @@
+# mobilefacedet-v1-mxnet
+
+## Use Case and High-Level Description
+
+  MobileFace Detection V1 is a Light and Fast Face Detector for Edge Devices (LFFD) model based on Yolo V3 architecture and trained with MXNet. For details see the [repository](https://github.com/becauseofAI/MobileFace) and [paper](https://arxiv.org/pdf/1904.10633.pdf).
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Detection     |
+| GFLOPs            | 3.5456       |
+| MParams           | 7.6828        |
+| Source framework  | MXNet\*       |
+
+## Input
+
+### Original model
+
+Image, name - `data`, shape - `[1x256x256x3]`, format -`[BxHxWxC]` where:
+
+- `B` - batch size
+- `H` - height
+- `W` - width
+- `C` - channel
+
+Expected color order -  `BGR`.
+
+### Converted model
+
+The converted model has the same parameters as the original model.
+
+## Output
+
+### Original model
+
+1. The array of detection summary info, name - `yolov30_slice_axis1`,  shape - `1,18,8,8`. The anchor values are `118,157,  186,248,  285,379`.
+
+2. The array of detection summary info, name - `yolov30_slice_axis2`,  shape - `1,18,16,16`. The anchor values are `43,54,  60,75,  80,106`.
+
+3. The array of detection summary info, name - `yolov30_slice_axis3`,  shape - `1,18,32,32`. The anchor values are `10,12,  16,20,  23,29`.
+
+For each case format is `B,N,Cx,Cy`, where
+    - `B` - batch size
+    - `N` - number of detection boxes for cell
+    - `Cx`, `Cy` - cell index
+
+Detection box has format [`x`,`y`,`h`,`w`,`conf`,`class`], where:
+- (`x`,`y`) - coordinates of box center, relative to cell
+- `h`,`w` - normalized height and width of box
+- `conf` - confidence of detection box
+- `class` - score for class
+
+### Converted model
+
+1. The array of detection summary info, name - `yolov30_yolooutputv30_conv0_fwd/YoloRegion`,  shape - `1,18,8,8`. The anchor values are `118,157,  186,248,  285,379`.
+
+2. The array of detection summary info, name - `yolov30_yolooutputv31_conv0_fwd/YoloRegion`,  shape - `1,18,16,16`. The anchor values are `43,54,  60,75,  80,106`.
+
+3. The array of detection summary info, name - `yolov30_yolooutputv32_conv0_fwd/YoloRegion`,  shape - `1,18,32,32`. The anchor values are `10,12,  16,20,  23,29`.
+
+For each case format is `B,N,Cx,Cy`, where
+    - `B` - batch size
+    - `N` - number of detection boxes for cell
+    - `Cx`, `Cy` - cell index
+
+Detection box has format [`x`,`y`,`h`,`w`,`conf`,`class`], where:
+- (`x`,`y`) - coordinates of box center, relative to cell
+- `h`,`w` - normalized height and width of box
+- `conf` - confidence of detection box
+- `class` - score for class in the [0,1] range
```

Comment:
What the difference between `score` and `confidence` ?

Reply:
It is not clear for me, the reason of this question... Could you please give more context?

Reply:
generally, `conf` is objectness score (there is object in this cell or not), `class` is confidence that detected object belongs to specific class.
The output format is the same like in Yolo V3, but model trained to detect objects for only 1 class - face

Reply:
Yes, I understand the difference, and even why here present both of them. But for the outside view it is not so obvious and may be confusing.

Also, do we need some postprocessing, like multiplying them to get correct score (like YOLO do)?

Reply:
@eizamaliev if I right understand, @Anna-Mironova just  tried to follow guidelines given in yolo-v3 model description, preserving the same info level and terminology (BTW, yolo-v3 description does not contains info about postprocessing). If you have suggestion how to improve it, please give. We will be appreciate to you help.

Reply:
@eaidova thanks for remark. I will expand descriptions of YOLOs and then help you.

Reply:
Please refer to [PR 1029](https://github.com/opencv/open_model_zoo/pull/1029) (also you are welcomed for review)

Reply:
My main point was what `conf`  is a probability of object (any type) and `classes` is a probability distribution over classes. And the real class confidence is multiplication of both. But for only one class it is pointless.

Reply:
@eizamaliev , @eaidova  thanks. please review again